test/e2e/openshift/util/util.go

```diff
@@ -153,6 +139,27 @@ func FetchLogs(kind, namespace, name string) string {
 	return string(out)
 }
 
+// FetchClusterInfo returns node and pod information about the cluster.
+func FetchClusterInfo(logPath string) {
+	needsLog := map[string]string{
```

Comment:
I'm happy for this to merge, but you don't think `map[string]func() error` would be a better foundation here?

Reply:
I don't feel strong either way, updated to your suggestion (which needs to be `map[string]func() string`).

Reply:
apols, I meant `map[string]func() (string, error)` - in which case the changes to the other functions aren't needed and they're more reusable

Reply:
ack, updated